ONSdigital/eq-author
# eq-355 author login

### What is the context of this PR?
Adds the basic setup needed to provide a login ui to the app with validation messaging.

### How to review 
1. Run the application and check the health page is still accessible without sign in (http://127.0.0.1:8000/healthcheck)
2. Go to http://127.0.0.1:8000/admin/ and login to the Django (probably admin, password)
2. Create a new user, usernames are in the format of emails
3. Logout of Django admin
4. Check you can't use the rest API without a token (http://127.0.0.1:8000/api/v1/schema/)
5. Now try to visit the root (http://127.0.0.1:8000/) it should redirect you to sign-in. You shall not pass
6. Fill the form in with a combinations or garbage to make sure it works
7. Now fill it in with the correct details you created earlier and submit, this should now allow you in
8. Now use the author to generate and see schemas
8. Sign out by going to http://127.0.0.1:8000/sign-out
9. Check you are signed out (http://127.0.0.1:8000/)


### Who worked on the PR

@hamishtaplin and @LJBabbage
